modules/exploits/linux/http/airties_login_cgi_bof.rb

```diff
@@ -0,0 +1,150 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
```

Comment:
* The CMD payloads are not working on this device.
* Executing from memory could be possible. In the past I had some troubles with the MIPS encoders  on different devices. There was a different behavior between emulation and the real device. The Stager was always working as expected.